islandora_paged_content_pdf_batch.drush.inc

```diff
@@ -69,7 +65,7 @@ function islandora_paged_content_pdf_batch_drush_command() {
       ),
       'namespace' => array(
         'description' => dt('The namespace for objects created by this command.  Defaults to namespace set in Fedora config.'),
-        'required' => FALSE,
+        'required' => TRUE,
```

Comment:
Thanks for the pull request! Just curious why you changed this to now be required?